pkg/bugzilla/client.go

```diff
@@ -44,14 +43,25 @@ func NewClient(user, pass, address string) Client {
 	}
 }
 
+type arguments struct {
```

Comment:
should have godoc, even though it's not public (even just like `arguments is the set of arguments for the "savedsearch" bugzilla RPC`)